semantic_conventions/scope/exporter/exporter.yaml

```
@@ -0,0 +1,30 @@
+groups:
+  - id: otel.scope
+    prefix: otel.scope
+    type: resource
+    brief: Attributes used by non OTLP exporters to represent OpenTelemetry Scope's concepts.
+    attributes:
+      - id: name
+        type: string
+        brief: The name of the instrumentation scope - `InstrumentationScope.Name`.
+        examples: ['io.opentelemetry.contrib.mongodb']
+      - id: version
+        type: string
+        brief: The version of the instrumentation scope - `InstrumentationScope.Version`.
```

```suggestion
        brief: The name of the instrumentation scope (`InstrumentationScope.Name` in OTLP).
        examples: ['io.opentelemetry.contrib.mongodb']
      - id: version
        type: string
        brief: The version of the instrumentation scope (`InstrumentationScope.Version` in OTLP).
```
Might be confusing what's meant here otherwise.

---

ok. done.